### PR TITLE
Adding Grape::Tokeeo

### PR DESCRIPTION
**Pull request by paganotoni**
Adding Grape::Tokeeo to the authentication section.

### CODE REVIEW

**Comment by paganotoni** [comment]
@dblock would this gem be ok for your docs ?

File: projects/index.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ comments: false
 * [grape-doorkeeper](https://github.com/fuCtor/grape-doorkeeper): Grape integration with Doorkeeper.
 * [grape_api_signature](https://github.com/faber-lotto/grape_api_signature): A RACK-Based AWS Style signature authentication for grape API.
 * [wine_bouncer](https://github.com/antek-drzewiecki/wine_bouncer): Doorkeeper OAuth2 authorization middleware for grape APIs.
-
+* [grape-tokeeo](https://github.com/wawandco/grape-tokeeo): A Simple Authentication gem for grape.
```
Comment (dblock) [inline projects/index.md]: Spell grape Grape please :) And make the empty line after your gem? Via `--amend`, thank you!

**Comment by paganotoni** [comment]
@dblock how about now ?

**Comment by dblock** [comment]
Great, can you squash the commits pls?

**Comment by paganotoni** [comment]
@dblock I hope this works, first time --amend'ing on a PR, lesson learned :)

**Comment by dblock** [comment]
Merged, thanks for having out there!

**Pull request merged**